components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/api/RecoveryApiV2.java

```diff
@@ -100,6 +106,54 @@ public RecoveryResponse recoverPassword(RecoveryRequest recoveryRequest, String
         return recover(recoveryRequest, tenantDomain, headers, localVarPath);
     }
 
+    /**
+     * Resend password recovery confirmation details.
+     *
+     * @param resendRequest   Resend request. (required)
+     * @param tenantDomain      Tenant Domain which user belongs. Default "carbon.super" (optional)
+     * @param headers           Any additional headers to be embedded. (optional)
+     * @return Resend response.
+     * @throws ApiException If fails to make API call.
+     */
+    public ResendResponse resendPasswordNotification(ResendRequest resendRequest, String tenantDomain,
+                                                     Map<String, String> headers) throws ApiException {
+
+        String localVarPath = "/password/resend".replaceAll("\\{format\\}", "json");
```

Comment:
why do we need to do a replaceAll for `password/resend` string? isnt this redundent?

Reply:
look for other places as well